test_data/indented_lambda.good.nix

```diff
@@ -8,15 +8,15 @@
     else "--${k}"
 , mkOption ? k: v:
     if v == null
-    then [ ]
-    else [ (mkOptionName k) (lib.generators.mkValueStringDefault { } v) ]
+    then []
+    else [ (mkOptionName k) (lib.generators.mkValueStringDefault {} v) ]
```

Comment:
This is a good example where I find the new rule to be less consistent. Why have spaces only on the second line?

Reply:
Move to the issue